test/parallel/test-fs-write-file-invalid-path.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+if (!common.isWindows)
```

Comment:
Maybe the skipping can be placed before `require('assert')` etc. to eliminate needless work.